File: guides/java/kora-java-guide-observability-app/src/main/java/ru/tinkoff/kora/guide/observability/health/ApplicationHealthProbe.java
package ru.tinkoff.kora.guide.observability.health;

import ru.tinkoff.kora.common.Component;
import ru.tinkoff.kora.common.liveness.LivenessProbe;
import ru.tinkoff.kora.common.liveness.LivenessProbeFailure;

@Component
public final class ApplicationHealthProbe implements LivenessProbe {

    @Override
    public LivenessProbeFailure probe() {
        return null;
    }
}

File: guides/kotlin/kora-kotlin-guide-observability-app/src/main/kotlin/ru/tinkoff/kora/guide/observability/health/ApplicationHealthProbe.kt
package ru.tinkoff.kora.guide.observability.health

import ru.tinkoff.kora.common.Component
import ru.tinkoff.kora.common.liveness.LivenessProbe
import ru.tinkoff.kora.common.liveness.LivenessProbeFailure

@Component
class ApplicationHealthProbe : LivenessProbe {
    override fun probe(): LivenessProbeFailure? = null
}
